src/marshal.c

```diff
@@ -242,6 +337,7 @@ int decode_packet(bot_t *bot, void *_packet_raw, void *_packet_data)
     char *packet_raw = _packet_raw;
     char *packet_data = _packet_data;
 
+    uint64_t bytes_read = 0; /* bytes read from packet_raw */
```

Comment:
What's the point of this variable? Would it be better to just do `packet_raw - _packet_raw`?